shared/eventbus/eventbus.go

```diff
@@ -150,7 +162,44 @@ func (b *BrokerService) runHandlers() {
 	for {
```

Comment:
Can we also add a way to exit this `for` loop? Such as:

```Go
for ev := range b.channel {
	...
}
```

Reply:
Oh, yes. That should be added.